packages/cli/src/commands/config/config.ts

```diff
@@ -26,7 +23,7 @@ function filterConfig(config) {
 export default {
   name: 'config',
   description: 'Print CLI configuration',
-  func: async (argv: string[], ctx: ConfigT) => {
+  func: async (ctx: Config) => {
```

Comment:
What happened to the first argument of the function? Have you made sure everything works as expected with this change?

Reply:
`Config` is a second argument. This change breaks that. If `argv` is not used, prepend it with an underscore: `_argv`

Reply:
OK. I din't notice that, I'll fix it.